pkg/lib/page.scss

```diff
@@ -75,6 +75,33 @@ a.disabled:hover {
     width: 100%;
 }
 
+/* Animation of new items */
+
+.new-item {
```

Comment:
Does this really have to go into the global page.css? I'd much rather like to see this in pkg/lib/cockpit-components-table.scss, to avoid namespace collisions. Otherwise please add a `ct-table` to the name. Same for `newRow`.

Reply:
I originally had it in `pkg/lib/cockpit-components-table.scss` but moved it to `pkg/lib/page.scss` as we want to use the same pattern on different components as well. As a good example see https://github.com/cockpit-project/cockpit/issues/10059 where we should also add animation but it does not use `ListingTable`. I purposefully want to create this as a global item so that we can just add this class into any page into any element and it just works the same way.

Reply:
I see, thanks! So let's please at least add a `-ct` suffix